src/Views.elm

```diff
@@ -16,12 +16,15 @@ taskItemView index task =
        onDragStart <| Move task,
        attribute "ondragstart" "event.dataTransfer.setData('text/plain', '')"
      ]
-     [ text task.name
+     [ enrichItemContent task.name
      , button [ class "btn-delete"
               , onClick <| Delete task.name
               ][ text "+" ]
      ]
 
+enrichItemContent : String -> Html Msg
+enrichItemContent str = Html.div [] (List.intersperse (text " ") (List.map (\w -> if String.startsWith "http" w then a [target "_blank", href w] [text w] else text w) (String.words str)))
```

Comment:
Just the coding style concern, would you mind rearrange this function for a bit more readability, please?